pokemon_battle_arena/src/db/pokemon_token.rs

```diff
@@ -125,20 +125,29 @@ impl PokemonToken {
         None
     }
 
-// TO-DO: Getter methoden für move bestimmt fehlerbehaftet (unwrap() Aufruf)
+
+    /// Getter function for move one. If the move is set, the function returns it, if not,
+    /// it returns None
     pub fn get_move_one(self) -> Option<moves::Technique> {
-        Some(self.move_one.unwrap().0)
+        Some(self.move_one.unwrap_or(return None).0)
```

Comment:
These won't work *at all*. [This function will *always* return `None`](https://is.gd/eCw2Pk). I think the compiler will even warn about this... you really shouldn't ignore warnings! I think you want to use `Option::map()` here. 

Getting familiar with several helper methods of `Option`, `Result` and `Iterator` will improve your code style and make programming easier. Don't *fight* Rust: get used to how Rust wants things to be done 😉 

You should probably use some kind of code review or at least Travis-CI to avoid a few of those bugs. 

(*You can answer in German, too. I'm not sure what language you use to discuss here on GitHub*)